index.html

```diff
@@ -1,24 +1,40 @@
-<link href="https://fonts.googleapis.com/css?family=Josefin+Sans:100&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:100&display=swap" rel="stylesheet">
 </head>
-
-      <header>
-            <nav>
-                 <li><a href="#home">Home</a></li>
-                 <li><a href="#projects_section">Projects</a></li>
-                  <li>Contact</li>
-             <nav>
-         </header>
+<header>
+  <nav>
+    <li><a href="#home">Home</a></li>
+    <li><a href="#projects_section">Projects</a></li>
+    <li><a href="#contact">Contact</a></li>
+   <nav>
+</header>
     
- <main id="home">
-    <h1 id= "welcome"> Hello! My name is Nicole, an aspiring web deveolper and web designer.</h1>
-    <h2 id="welcome_subtitle"> Welcome to my portfolio, feel free to shuffle through and see what you like!</h2>
+<main id="home">
+  <h1 class= "welcome"> Hello! My name is Nicole, an aspiring web deveolper and web designer.</h1>
+  <h2 class="welcome_subtitle"> Welcome to my portfolio, feel free to shuffle through and see what you like!</h2>
 </main>
     
-<div id= "projects_section">
+<div id= "projects_section" class= "projects_section">
   
-    <h2 id="project_opener"> Here are some sites I've created...</h2>
+  <h2 class="project_opener"> Here are some sites I've created...</h2>
   
-    <p id= "project_text" class= "project_opener">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic ...</p>
-    </div>
\ No newline at end of file
+  <p class= "project_text" class= "project_opener">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic ...</p>
+  
+  <h2 class= "checklist"> To-Do List</h2>
+  <img class="projectpic" src="https://images.unsplash.com/photo-1547480053-7d174f67b557?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80" alt="Check list on a desk">
+       
+  <h2 class= "recipe">Cooky Cookie's Recipe Card</h2>
+  <img class="projectpic" src="https://images.unsplash.com/photo-1512058454905-6b841e7ad132?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1586&q=80" alt="Recipe on counter">
+    
+  <h2 class= "timer"> Tabata Timer</h2>
+  <img class="projectpic" src="https://images.unsplash.com/photo-1431499012454-31a9601150c9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80" alt="Timer in hand">
+</div>
+    
+<div id= "contact" class= "contact">
+  <h2>Email: email@web.com</h2>
+  <h2>Phone: 888-888-8888</h2>
```

Comment:
leave your phone number off the site. but also use a tele link instead of a heading